xonsh/shells/ptk_shell/key_bindings.py

```diff
@@ -428,6 +432,23 @@ def create_alias(new_keys, original_keys):
 
     # Complete a single auto-suggestion word
     create_alias([Keys.ControlRight], ["escape", "f"])
+    create_alias([Keys.ControlSquareClose], ["escape", "f"])
+
+    @handle(Keys.ControlRight, filter=has_suggestion)
+    @handle(Keys.ControlSquareClose, filter=has_suggestion)
```

Comment:
A note, and I'm not sure why this is the case, `@handle(Keys.ControlSquareClose, filter=has_suggestion)` only is applying in VI Mode, and not Emacs modes like I'd expect.